src/python/twitter/common/reviewboard/reviewboard.py

```diff
@@ -103,7 +103,7 @@ def _add_auth_params(self, username, password):
     urllib2.install_opener(opener)
 
   def get_url(self, rb_id):
-    return '%s/r/%s' % (self.url, rb_id)
+    return '%sr/%s' % (self.url, rb_id)
```

Comment:
Wait

```py
>>> "%sr/%s" % (url, "fungus")
'https://foo.host/bar/r/fungus'
>>> url + "r/%s" % ("fungus",)
'https://foo.host/bar/r/fungus'
```
you need to put a ',' in your tuple when using % interpolation. the ',' is what makes it a tuple, not the parens.